packages/elements/src/components/ino-select/ino-select.tsx

```diff
@@ -71,11 +73,35 @@ export class Select implements ComponentInterface {
    */
   @Event() valueChange!: EventEmitter<string>;
 
+  /**
+   * Option Values to display inside hidden <select></select>
+   *
+   * @type {Array<string>}
+   * @memberof Select
+   */
+  @State() mdcSelectMenuListOptionValues: Array<{
+    value: string;
+    selected: boolean;
+    disabled: boolean;
+  }> = [];
+
+  componentWillLoad() {}
+
   componentDidLoad() {
     this.mdcSelectInstance = new MDCSelect(
       this.el.querySelector('.mdc-select')
     );
 
+    // extract ino-options to create hidden a11y select element <select><option></option>/select>
+    const mdcSelectMenuList = this.mdcSelectMenuWrapper.querySelector('ul');
+    this.mdcSelectMenuListOptionValues = [
```

Comment:
We should have in mind that the `ino-option`s could be changed dynamically and apply this logic then.

Reply:
This does not need to be fixed in this context. I opened a ticket: https://github.com/inovex/elements/issues/366